keyboards/telophase/rules.mk

```diff
@@ -1,6 +1,10 @@
 # MCU name
 MCU = atmega32u4
 
+# Processor frequency
+F_CPU = 8000000
+
```

Comment:
If this is using a Pro Micro, I think this should actually be removed, as the 16MHz variant is far more common. Users building one of these with an 8MHz Pro Micro can just add it to their keymap's rules.mk.